### PR TITLE
[CP] Katello 3.13.3 cherry-picks, updated changelog, and version bump.

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# 3.13.3 Baltic Porter (2019-12-02)
+
+## Features
```
Comment (jturel) [inline CHANGELOG.md]: Unnecessary features heading here, otherwise LGTM!

**Comment by jjeffers** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

File: test/actions/katello/event_queue_monitor_test.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ class Actions::Katello::EventQueueMonitorTest < ActiveSupport::TestCase
     let(:suspended_class) { ::Actions::Katello::EventQueue::SuspendedAction }
     let(:planned_action) do
       action_class.any_instance.expects(:singleton_lock!)
-      action_class.any_instance.expects(:holds_singleton_lock?).returns(true)
+      action_class.any_instance.expects(:holds_singleton_lock?).returns
```
Comment (jturel) [inline test/actions/katello/event_queue_monitor_test.rb]: This change should be on the line just above :)

Comment (jjeffers) [reply test/actions/katello/event_queue_monitor_test.rb]: Mea culpa.

**Pull request merged**